pkg/volume/csi/csi_plugin.go

```diff
@@ -54,7 +54,7 @@ const (
 	// TODO (vladimirvivien) would be nice to name socket with a .sock extension
 	// for consistency.
 	csiAddrTemplate = "/var/lib/kubelet/plugins/%v/csi.sock"
-	csiTimeout      = 15 * time.Second
+	csiTimeout      = 2 * time.Minute
```

Comment:
Not sure what is the best default value. If the csiTimeout can be configurable like other csi sidecar do that will be better.

Reply:
@hoyho yes, but now I would like cherry-pick this fix to old release from 1.13 to 1.15, 15s is really too small, it makes the timeout a lot. And when user switch to use CSI driver, they won't be happy to  see that timeout issue every time.

Reply:
agree with you. For our CSI driver, 15s is not enough.

Reply:
I also meet some issues caused by the default time. 

LGTM